tick/inference/base/learner_glm.py

```diff
@@ -53,21 +53,29 @@ class LearnerGLM(LearnerOptim):
         Record history information when ``n_iter`` (iteration number) is
         a multiple of ``record_every``
 
+    random_state : int seed, RandomState instance, or None (default)
+        The seed that will be used by stochastic solvers. Used in 'sgd',
+        'svrg', and 'sdca' solvers
+        
+    Other Parameters
+    ----------------
     sdca_ridge_strength : `float`, default=1e-3
         It controls the strength of the additional ridge penalization. Used in
         'sdca' solver
 
     elastic_net_ratio : `float`, default=0.95
         Ratio of elastic net mixing parameter with 0 <= ratio <= 1.
-        For ratio = 0 this is ridge (L2) regularization
+        For ratio = 0 this is ridge (L2 squared) regularization
         For ratio = 1 this is lasso (L1) regularization
         For 0 < ratio < 1, the regularization is a linear combination
         of L1 and L2.
         Used in 'elasticnet' penalty
-
-    random_state : int seed, RandomState instance, or None (default)
-        The seed that will be used by stochastic solvers. Used in 'sgd',
-        'svrg', and 'sdca' solvers
+        
+    blocks_start : `numpy.array`, shape=(n_features,), default=None
+        The indices of the first column of each binarized feature blocks
+        
+    blocks_length : `numpy.array`, shape=(n_features,), default=None
+        The length of each binarized feature blocks
```

Comment:
Someone who is using learner does not know much about prox binarsity. You should be very explicit about these two parameters and specify that they will only be used for prox binarsity